internal/provider/provider.go

```diff
@@ -49,6 +49,12 @@ func New(version string) func() *schema.Provider {
 					Default:     "",
 					Type:        schema.TypeString,
 				},
+				BaseImageKey: {
+					Description: "Default base image for builds",
+					Optional:    true,
+					Default:     "",
```

Comment:
Should this be static:latest-glibc?

Reply:
If we don't set anything then this should fall through to ko defaults, which is `cgr.dev/chainguard/static:latest` (equiv to what was being set before): https://github.com/ko-build/ko/blob/1f6a357d8f00a701126c5d97bc8eaf2858d5e73b/pkg/commands/options/build.go#L99

So maybe? Idk how prescriptive we want to be in terraform vs deferring to ko.

Reply:
I'll defer to you here - i'm fine with either.